README.adoc

```diff
@@ -1,8 +1,8 @@
 = Spring Cloud Data Flow Server for Cloud Foundry image:https://build.spring.io/plugins/servlet/buildStatusImage/SCD-CFBMASTER[Build Status, link=https://build.spring.io/browse/SCD-CFBMASTER] image:https://badge.waffle.io/spring-cloud/spring-cloud-dataflow-server-cloudfoundry.svg?label=ready&title=Ready[Stories in Ready, link=http://waffle.io/spring-cloud/spring-cloud-dataflow-server-cloudfoundry] image:https://badge.waffle.io/spring-cloud/spring-cloud-dataflow-server-cloudfoundry.svg?label=In%20Progress&title=In%20Progress[Stories in Progress, link=http://waffle.io/spring-cloud/spring-cloud-dataflow-server-cloudfoundry]
 
-This project provides support for deploying https://github.com/spring-cloud/spring-cloud-dataflow[Spring Cloud Dataflow] Stream definitions to Cloud Foundry.  It includes an implementation of the https://github.com/spring-cloud/spring-cloud-deployer[Spring Cloud Dataflow Deployer SPI] for Cloud Foundry.
+This project provides support for deploying https://github.com/spring-cloud/spring-cloud-dataflow[Spring Cloud Data Flow]'s streaming data pipelines to Cloud Foundry. It includes an implementation of Spring Cloud Data Flow's https://github.com/spring-cloud/spring-cloud-deployer[Deployer SPI] for Cloud Foundry.
 
-Please refer to the http://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started[reference documentation] on how to get started.
+Please refer to the http://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/current-SNAPSHOT/reference/htmlsingle/#index[reference documentation] on how to get started.
```

Comment:
This needs to be a variable defined in index.adoc so we can go to one spot and change it for the release vs. everywhere it might be used.  See https://raw.githubusercontent.com/spring-cloud/spring-cloud-stream/master/spring-cloud-stream-docs/src/main/asciidoc/index.adoc as an example